spring-boot-project/spring-boot-docs/src/docs/asciidoc/messaging/pulsar.adoc

```diff
@@ -0,0 +1,144 @@
+[[messaging.pulsar]]
+== Apache Pulsar Support
+https://pulsar.apache.org/[Apache Pulsar] is supported by providing auto-configuration of the {spring-pulsar-docs}[Spring for Apache Pulsar] project.
+
+Spring Boot will auto-configure and register the classic (imperative) Spring Pulsar components when `org.springframework.pulsar:spring-pulsar` is on the classpath.
+It will do the same for the reactive components when both `org.springframework.pulsar:spring-pulsar-reactive` and Project Reactor are on the classpath.
+
+There are `spring-boot-starter-pulsar` and `spring-boot-starter-pulsar-reactive` "`Starters`" for conveniently collecting the dependencies for imperative and reactive use, respectively.
+
+TIP: The Reactive support is built on top of the imperative support and therefore the reactive starter includes the imperative starter.
+As such, the imperative components are auto-configured and available in both reactive and imperative scenarios.
```

Comment:
What happens if you use the imperative components in a reactive app? Will the blocking cause a problem?